cmd/eib/main.go

```diff
@@ -52,16 +44,26 @@ func processArgs() (*config.ImageConfig, *config.BuildConfig, error) {
 	}
 	buildConfig := config.BuildConfig{
 		ImageConfigDir: configDir,
-		BuildDir: buildDir,
+		BuildDir:       buildDir,
 	}
 
 	return imageConfig, &buildConfig, err
 }
 
-func handleVerbose(verbose bool) {
+func setupLogging(verbose bool) {
+	logLevel := zap.InfoLevel
 	if verbose {
-		log.SetLevel(log.DebugLevel)
+		logLevel = zap.DebugLevel
+	}
+
+	logConfig := zap.Config{
+		Level: zap.NewAtomicLevelAt(logLevel),
 	}
+
+	logger := zap.Must(logConfig.Build())
```

Comment:
I tested it and it panics for me with:
```panic: no encoder name specified```

If I set that up, it no longer panics but still doesn't log anything. Have you tried it?

Reply:
Oh shit, I completely misread the error message. I tried it simulating an error so that I'd see the logs and it looked like the error was what I expected.